pkgs/applications/misc/with-shell/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation {
+  name = "with";
```

Comment:
If a package is install from git and does not have a version, you better use the date of the commit as version:
```
name = "with-2016-08-20";
```